src/sentry/api/bases/team.py

```diff
@@ -52,7 +52,7 @@ def convert_args(self, request: Request, organization_slug, team_slug, *args, **
 
         bind_organization_context(team.organization)
 
-        request._request.organization = team.organization
+        request._request.organization = team.organization  # type: ignore[attr-defined]
```

Comment:
we should not be stuffing random things into request

can you make sure nothing reads this and delete it?

Reply:
yes, will add it to the TODO!